### DISCUSSION TITLE
speakeasy

### DISCUSSION THREAD

**Discussion by leedsleeds** [Security-Onion-Solutions/securityonion#4306] [Unsupported Versions] [answered]
Hey

I have downloaded the latest version of Security Onion (2.3.51) & generated an AV alert. While it looks fine after investigating, would anyone point me in the direction needed to understand where Speakeasy is utilized within SO?

My understanding is "Speakeasy is a portable, modular, binary emulator designed to emulate Windows kernel and user mode malware", so would make sense why it triggers an AV alert but not sure which competent of SO uses this?

![image](https://user-images.githubusercontent.com/61006442/119812885-8b108700-bee0-11eb-8e37-ce5183cd2e10.png)

**Comment by dougburks** [comment, marked as answer]
Speakeasy is part of Strelka:
https://docs.securityonion.net/en/2.3/strelka.html

**Reply by leedsleeds** [reply]
Thanks :)

**Reply by jefferymcaron** [reply]
Has this finding been registered and confirmed with DISA?

Thanks!
